data-processing/Dockerfile

```diff
@@ -35,35 +45,26 @@ RUN python get-pip.py --force-reinstall
 RUN curl -sL https://deb.nodesource.com/setup_12.x | bash
 RUN apt-get install -y nodejs
 
-# Let ImageMagick to read and write PDFs
-RUN sed -i '/pattern="PDF"/s/rights="none"/rights="read | write"/' /etc/ImageMagick-6/policy.xml
```

Comment:
Do we not need this anymore?

Reply:
Talked to Andrew and likely we don't need this anymore

Reply:
Turns out, we still need this. Latex compilation fails without this.